### PR TITLE
Ziggo mediabox xl

### PR DESCRIPTION
**Pull request by b10m**
**Description:**
Support page + image for the new ziggo mediabox xl media_player component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#10514

## Checklist:

  - [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/media_player.ziggo_mediabox_xl.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+description: "Instructions how to integrate the Ziggo mediabox into Home Assistant."
+date: 2017-11-10 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ziggo.png
+ha_category: Media Player
+ha_iot_class: "Local Push"
+ha_release: "0.58"
```
Comment (frenck) [inline source/_components/media_player.ziggo_mediabox_xl.markdown]: `0.58` is already shipped. You might want to update this one.

File: source/_components/media_player.ziggo_mediabox_xl.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: "Ziggo Mediabox XL"
+description: "Instructions how to integrate the Ziggo mediabox into Home Assistant."
```
Comment (frenck) [inline source/_components/media_player.ziggo_mediabox_xl.markdown]: Grammar: `Instructions on how to`
Typo: `Ziggo mediabox` -> `Ziggo Mediabox`

File: source/_components/media_player.ziggo_mediabox_xl.markdown [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+comments: false
+sharing: true
+footer: true
+logo: ziggo.png
+ha_category: Media Player
+ha_iot_class: "Local Push"
+ha_release: "0.58"
+---
+
+The `Ziggo Mediabox XL` component allows you to control a [Ziggo](https://www.ziggo.nl/) mediabox from Home Assistant.
```
Comment (frenck) [inline source/_components/media_player.ziggo_mediabox_xl.markdown]: Typo: `[Ziggo](https://www.ziggo.nl/) mediabox` -> `[Ziggo](https://www.ziggo.nl/) Mediabox`
Lower case component: `Ziggo Mediabox XL` -> `ziggo_mediabox_xl`

File: source/_components/media_player.ziggo_mediabox_xl.markdown [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+# Example configuration.yaml entry
+media_player:
+  - platform: ziggo_mediabox_xl
+    host: 192.168.0.123
+    name: Ziggo Mediabox
+```
+
+{% configuration %}
+  host:
+    description: The host name or address of the device.
```
Comment (frenck) [inline source/_components/media_player.ziggo_mediabox_xl.markdown]: Typo: `host name` -> `hostname`

File: source/_components/media_player.ziggo_mediabox_xl.markdown [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    description: The name of the device used in the frontend.
+    required: false
+    type: string
+{% endconfiguration %}
+
+The channel information (numbers and names) are downloaded from ziggo.nl on startup.
+
+#### {% linkable_title Preparation of the Mediabox %}
+
+Makes sure to enable the Home Network ("mijn thuisnetwerk") service in the settings menu of the mediabox. Once you have set up the Media Library ("mediabiblotheek"), we can determine whether the device is turned on or off. Without this, the component will fail to start.
```
Comment (frenck) [inline source/_components/media_player.ziggo_mediabox_xl.markdown]: If you want to refer to the `mediabox` word in general: `mediabox` -> `media box`.
When you are referring to one Ziggo provides: `mediabox` -> `Ziggo Mediabox`

**Comment by frenck** [comment]
@b10m Thanks!
This PR can be merged as soon as the parent PR gets merged.

File: source/_components/media_player.ziggo_mediabox_xl.markdown [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+title: "Ziggo Mediabox XL"
+description: "Instructions how to integrate the Ziggo Mediabox XL into Home Assistant."
+date: 2017-11-10 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ziggo.png
+ha_category: Media Player
+ha_iot_class: "Local Push"
```
Comment (MartinHjelmare) [inline source/_components/media_player.ziggo_mediabox_xl.markdown]: "Local Polling"

**Pull request merged**